docs/src/testing/load_testing/load_testing.md

```diff
@@ -35,38 +44,24 @@ And paste this:
 ```conf 
 Host myrra_load_test_client
   Hostname client_ip
+  User user
 
 Host myrra_load_test_server
   Hostname game_ip
+  User user
 ```
 
-### Game Server Setup
-1. Check you can log into it with ssh: 
-   ```sh
-   ssh myuser@myrra_load_test_server
-   ```
-2. If it's not already there exit, copy the script on this repo under
-   `game_backend/load_test/setup_game_server.sh` it clones the game server and compiles it:
-   ```sh
-   scp /path_go_game_backend/game_backend/load_test/setup_game_server.sh myrra_load_test_server:/user/setup_game_server.sh
-   ```
-   Then relog (step 1) and relog into the server 
-   `setup_game_server` can also take a branch name as an argument. So if you want to run the load test on an specific branch, you can instead do:
-   ```sh
-   chmod +x ./setup_game_server.sh && ./setup_game_server.sh <BRANCH_NAME_TO_TEST>
-   ```
+## Running Load tests
 
-3. Now you can start the game server with: 
-```sh
-export $(cat .env | xargs) && cd game_backend && MIX_ENV=prod iex -S mix phx.server
-```
-   You can check the logs with `journalctl -xefu curse_of_myrra`.
-   From now on, you can just use: 
-```sh
-MIX_ENV=prod iex -S mix phx.server
-```
-   
-4. Make sure to disable hyperthreading, if using an x86 CPU:
+You're going to need 3 tabs in order to run load tests: 
+
+- one for the load test server
+- one to monitor the load test server status
+- one for the load test runner
```

Comment:
What do you think about clarifying in which server each tab belongs? something like 
```suggestion
You're going to need 3 tabs in order to run load tests: 
on the server ip: 
    - one for the load test server
    - one to monitor the load test server status (e.g. htop)
On the client ip:
   - load test runner
```

Reply:
Looks good!